hackathon_site/registration/models.py

```diff
@@ -99,19 +106,63 @@ class Application(models.Model):
         ],
         null=False,
     )
-    q1 = models.TextField(null=False, help_text="First question?", max_length=1000)
-    q2 = models.TextField(null=False, help_text="Second question?", max_length=1000)
-    q3 = models.TextField(null=False, help_text="Third question?", max_length=1000)
+    linkedin = models.URLField(
+        max_length=200, help_text="LinkedIn Profile", null=True, blank=True
+    )
+    github = models.URLField(
+        max_length=200, help_text="Github Profile", null=True, blank=True
+    )
+    devpost = models.URLField(
+        max_length=200, help_text="Devpost Profile", null=True, blank=True
+    )
+    why_participate = models.TextField(
+        null=False,
+        help_text="Why do you want to participate in NewHacks? (1000 char. max)",
+        max_length=1000,
+    )
+    what_technical_experience = models.TextField(
+        null=False,
+        help_text="What is your technical experience with software? (1000 char. max)",
+        max_length=1000,
+    )
+    what_past_experience = models.TextField(
+        null=False,
+        help_text="If you’ve been to a hackathon, briefly tell us your experience. If not, describe what you expect to see and experience. (1000 char. max)",
+        max_length=1000,
+    )
     conduct_agree = models.BooleanField(
-        help_text="I have read and agree to the code of conduct.",
+        help_text="I have read and agree to the "
+        '<a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH code of conduct</a>.',
```

Comment:
I forgot to mention this before but could you add `rel="noopener noreferrer" target="_blank"` to all the <a> tags so the links open in new tabs